README.md

```diff
@@ -10,84 +10,77 @@ the [Stylo](https://public.etherpad-mozilla.org/p/stylo) project.
 
 ## Requirements
 
-The current generator runs on with clang 3.8, but can also run with clang 3.9
-with more features (such as detection of inlined functions).
+It is recommended to use clang 3.9 with the current generator. It can run with
+clang 3.8 with some features unavailable.
 
-### Installing clang 3.8
+### Installing Clang 3.9
 
-#### OSX
+#### Windows
 
-```
-# brew install llvm38
-```
+Download and install the official pre-built binary from
+[LLVM download page](http://releases.llvm.org/download.html).
 
-#### On Debian-based Linuxes
+#### OSX
 
+If you use Homebrew:
 ```
-# apt-get install llvm-3.8-dev libclang-3.8-dev
+$ brew install llvm
 ```
-Adding the LLVM repos to get version 3.8 may be necessary, see http://apt.llvm.org/.
-#### Arch
 
+If you use MacPorts:
 ```
-# pacman -S clang clang-tools-extra
+$ port install clang-3.9
 ```
 
-### Getting clang 3.9
-
-#### From a package manager
-
-Clang 3.9 has ben released about a month ago, and some package managers already
-provide it.
-
-For example, for MacPorts:
+#### Debian-based Linuxes
 
 ```
-$ port install clang-3.9
-$ LIBCLANG_PATH=/opt/local/libexec/llvm-3.9/lib \
-  LD_LIBRARY_PATH=/opt/local/libexec/llvm-3.9/lib \
-  cargo build
+# apt-get install llvm-3.9-dev libclang-3.9-dev
 ```
 
-#### From source
-
-If your package manager doesn't yet offer Clang 3.9, you'll need to build from
-source. For that, follow the instructions
-[here](http://clang.llvm.org/get_started.html).
+Ubuntu 16.10 provides the necessary packages directly. If you are using older
+version of Ubuntu or other Debian-based distros, you may need to add the LLVM
+repos to get version 3.9. See http://apt.llvm.org/.
 
-Those instructions list optional steps. For bindgen:
+#### Arch
 
-* Checkout and build clang
-* Checkout and build the extra-clang-tools
-* Checkout and build the compiler-rt
-* You do not need to checkout or build libcxx
+```
+# pacman -S clang
+```
 
 ## Building
 
 ```
-$ cargo build --features llvm_stable
+$ cd bindgen
+$ cargo build
 ```
 
-If you want a build with extra features (llvm 3.9) then you can use:
-
+If you installed multiple versions of llvm, it may not be able to locate the
+latest version of libclang. In that case, you may want to either uninstall
+other versions of llvm, or specify the path of the desired libclang explicitly:
 ```
-$ LIBCLANG_PATH=path/to/clang-3.9/build/lib \
-  LD_LIBRARY_PATH=path/to/clang-3.9/build/lib \
-    cargo build
+$ export LIBCLANG_PATH=path/to/clang-3.9/lib
```

Comment:
Hmm, per description in clang-sys's README, it seems we would still need `LD_LIBRARY_PATH`.